data/h3_raster_importer/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.8"
+services:
+    landgriffon-seed-data:
+        build:
+            context: ./
+            dockerfile: Dockerfile
+        volumes:
+            - '../data/seed:/data'
+        command: make
+        env_file:
+            - '../../.env'
+        environment:
```

Comment:
Are these env vars required? 
I think you are getting values straight from the env file via

`psycopg2.connect(
        host=os.getenv('API_POSTGRES_HOST'),
        port=os.getenv('API_POSTGRES_PORT'),
        user=os.getenv('API_POSTGRES_USERNAME'),
        password=os.getenv('API_POSTGRES_PASSWORD')
    )`
    
    I don't see them anywhere else, but I could be missing something

Reply:
I don't recall how this works, but I suspect you may be right. we should test this, and remove the "environment" section if it's not needed